src/test/java/com/rocketden/main/socket/GameSocketTests.java

```diff
@@ -147,6 +165,20 @@ public void socketReceivesMessageOnGameOver() throws Exception {
         assertTrue(gameDto.getGameTimer().isTimeUp());
     }
 
+    @Test
+    public void socketReceivesNotificationMessageTenSecondsLeft() throws Exception {
+        GameNotificationDto notificationDto = notificationBlockingQueue.poll(TIME_UNTIL_TEN_LEFT + 5, SECONDS);
+
+        assertNotNull(notificationDto);
+        
+        assertNull(notificationDto.getInitiator());
+        assertEquals(NotificationType.TIME_LEFT,
+            notificationDto.getNotificationType());
+        assertTrue(LocalDateTime.now().isAfter(notificationDto.getTime())
+            || LocalDateTime.now().minusSeconds((long) 1).isBefore(notificationDto.getTime()));
```

Comment:
Just curious what this is checking?

Reply:
It's checking that the `notificationDto`'s time, since the time is set internally, is between 1 seconds before or after this moment. If you do an exact check the time's don't match, as the `notificationDto` was set up some number of milliseconds ago.